testcomponent/steps.go

```diff
@@ -35,13 +41,15 @@ type MongoV2Component struct {
 
 func (m *MongoV2Component) RegisterSteps(ctx *godog.ScenarioContext) {
 	ctx.Step(`^I have inserted these Records$`, m.insertedTheseRecords)
-	ctx.Step(`^I should receive these records$`, m.shouldReceiveTheseRecords)
+	ctx.Step(`^I should find these records$`, m.shouldReceiveTheseRecords)
+	ctx.Step(`^I should find no records, just a total count of (\d+)$`, m.shouldReceiveNoRecords)
 	ctx.Step(`^I will count (\d+) records$`, m.countRecords)
-	ctx.Step(`^I start a find operation`, m.findRecords)
+	ctx.Step(`^I filter on all records$`, m.findRecords)
 	ctx.Step(`^I set the limit to (\d+)`, m.setLimit)
 	ctx.Step(`^I skip (\d+) records$`, m.setSkip)
-	ctx.Step(`^I find records with Id > (\d+)$`, m.findWithId)
-	ctx.Step(`^I find this one record$`, m.findOneRecord)
+	ctx.Step(`^I set the ignore zero limit option$`, m.setIgnoreZeroLimit)
+	ctx.Step(`^I filter on records with Id > (\d+)$`, m.findWithId)
+	ctx.Step(`^find one should give me this one record$`, m.findOneRecord)
```

Comment:
To be consistent with the other steps, we should use capitals

```suggestion
	ctx.Step(`^Find One should give me this one record$`, m.findOneRecord)
```

Or even better change them both (lines 52 and 68) to `FindOne`?

```suggestion
	ctx.Step(`^FindOne should give me this one record$`, m.findOneRecord)
```

Reply:
Done